extension.js

```diff
@@ -69,7 +69,7 @@ const getWorkspaceRoot = doc => {
 function fullEdit(type, doc, formattingOptions) {
   let name = doc.fileName;
   let base = getWorkspaceRoot(doc) || vscode.workspace.rootPath || '';
-  let ignore = vscode.workspace.getConfiguration('beautify')
+  let ignore = vscode.workspace.getConfiguration('beautify',doc.uri)
```

Comment:
space after `,`.

Reply:
hehehe, i guess it was a little dumb of me to miss something like that on a code beautification project